ui/src/app/components/html-viewer.tsx

```diff
@@ -4,9 +4,13 @@ import { SpinnerOverlay } from 'app/components/spinners';
 import { withWindowSize } from 'app/utils';
 import { WindowSizeProps } from 'app/utils';
 
+// for query selection of the last document element
+const MS_WORD_PARAGRAPH_CLASS = '.MsoNormal';
```

Comment:
this is a good default because all of our current HTML docs (just TOS and DUCC) are generated by MS Word.